### ISSUE TITLE
Mute on load?

### ISSUE THREAD

**Issue by TylerLoch** [BashVideo/google-meet-push-to-talk#2]
We're hoping to deploy this to our students shortly! A great frictionless experience.
Any chance that this extension could toggle Mute on immediately upon joining a Meet?
Anything to reduce the chatter on a large video conference is welcome. 
Thank you!

**Comment by stopdropandrew** [comment]
@TylerLoch sure I think participants should be automatically muted after a certain number, five maybe, have joined -- that might be a GSuite setting. 

Either way, I can add it to the extension to cover the early joiners as well. I can get to it in the next day or two.

**Comment by TylerLoch** [comment]
Thank you!
And I can certainly see this being a subjective feature. Possibly something configurable via the admin console's Policy for extensions, along the lines of if "muteOnJoin" is true?

**Comment by stopdropandrew** [comment]
I can make it a setting. Do administrators have the ability to configure
settings from the admin console? If there's some documentation I can try to
work with that.

**Comment by TylerLoch** [comment]
Here's the documentation I've found on the process so far:
https://stackoverflow.com/questions/53256084/centralized-configuration-of-settings-for-a-google-chrome-extension
https://www.chromium.org/administrators/configuring-policy-for-extensions
https://developer.chrome.com/extensions/manifest/storage

Seems you must have a managed_schema entry in your manifest.json file.

And a screenshot where admins can add JSON entries in the G Suite Admin Console (Device management > Chrome > Apps & extensions):
https://imgur.com/cKgyZVH

**Comment by stopdropandrew** [comment]
@TylerLoch I set the default to mute on join with a configuration to disable it.

I looked into the managed storage options, but I couldn't get the directions from https://www.chromium.org/administrators/configuring-policy-for-extensions for testing locally to work on my mac laptop.

If there is interest in getting that set up, where the policy would override (and hide) the mute on join use option, I could try on a windows or linux machine, the instructions there were more straightforward.

Let me know if you have any other feature requests or ideas, I'm happy to help if I can!
